.github/workflows/linked-issue-check.yml

```
@@ -0,0 +1,129 @@
+#
+# Copyright the original author or authors from the JHipster project.
+#
+# This file is part of the JHipster project, see https://www.jhipster.tech/
+# for more information.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Linked Issue Check
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  linked-issue-check:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: read
+      pull-requests: write
```

I don't think we should add comment to PR that are missing closing issue.
We should keep actions permission as lower as possible.
```suggestion
      pull-requests: read
```